### DISCUSSION TITLE
Mobi to PK-Sim compatibility

### DISCUSSION THREAD

**Discussion by akim2000** [Open-Systems-Pharmacology/Forum#1627] [Questions & Problems]
If I were to create a detailed model of the brain compartment in MoBi, is it not possible to bring that model back to PK-Sim to use as a basis for normal PK-Sim procedures (i.e., using PK-Sim building blocks and such)? I think I read that you can only bring back a simulation from MoBi, so does that mean that every time I want to change something, I would need to make that change directly in MoBi, simulate it, then export the newly changed simulation to PK-Sim? Is there no way to replace the existing brain structure that PK-Sim uses via MoBi?

**Comment by PavelBal** [comment]
Unfortunately, this is not possible.
